libs/core/src/lib/info-label/info-label.component.spec.ts

```diff
@@ -36,23 +36,23 @@ describe('InfoLabelComponent', () => {
 
   it('Should Add label Type', () => {
     component.ngOnInit();
-    component.labelType = 'numeric';
+    component.type = 'numeric';
     component.ngOnChanges();
```

Comment:
You dont need to call ngOnChange, if you do and something does not there is could be a problem

Reply:
In general I agree on that but this is a unit test and ngOnChanges only rebuilds css classes, ofc there is `buildCssClass` which can be used